scripts/run_e2e_tests.py

```diff
@@ -130,17 +137,9 @@
 ]
 
 SUITES_STILL_IN_PROTRACTOR = [
-    'embedding',
-    'explorationImprovementsTab',
-    'explorationFeedbackTab',
-    'explorationHistoryTab',
-    'explorationStatisticsTab',
-    'explorationTranslationTab',
-    'extensions',
     'featureGating',
     'fileUploadFeatures',
     'fileUploadExtensions',
-    'learner',
```

Comment:
Is this just something we forgot to remove from earlier?

Reply:
Yes, learner is present in 'SUITE_MIGRATED_TO_WEBDRIVERIO'  but I forgot to remove this from 'SUITES_STILL_IN_PROTRACTOR'.
It was still working fine because while running the test we generally check whether the requested suite is present in webdriverio first.

Reply:
OK in that case, can we make the test stricter to catch these kinds of situations?

Reply:
Yes we can, but as the next migration PR will remove these checks because all the suites will be migrated, so I don't think adding test at this stage is needed now (Also I have confirmed now the list is not containing any duplicates).